packages/console/src/pages/Applications/hooks/use-legacy-application-data.ts

```diff
@@ -0,0 +1,59 @@
+import { type Application } from '@logto/schemas';
+import { useCallback, useMemo } from 'react';
+import useSWR from 'swr';
+
+import { type RequestError } from '@/hooks/use-api';
+import useSearchParametersWatcher from '@/hooks/use-search-parameters-watcher';
+import { buildUrl } from '@/utils/url';
+
+const pageSize = 2;
```

Comment:
```suggestion
const pageSize = 20;
```

Reply:
should be defaultPageSize, 2 is to testing use.